### PR TITLE
Final PR

### PR DESCRIPTION
**Pull request by MattGomb**
Dear Reviewer,

gif

This is the Final PR for the [Final Capstone Project](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md).

This PR is for the backend repository, which is an API-only Rails app. The readme for the backend includes the snapshot of the kanban board, and in contains the link to the [frontend repository](https://github.com/jfoyarzo/final-capstone-frontend).

Our group has 5 members @CRClothier, @emmiiorji, @iAmWaris97, @jfoyarzo, and @MattGomb.

### CODE REVIEW

**Review by Meltrust — COMMENTED** [review]
_note:  While performing this review, the PR was suddenly merged and closed.  After this, they opened another PR.  Consequently, I tried to communicate in Slack with the whole team to investigate what happened but got no answer at the time.  Reviewing this PR normally, as originally requested._

Hi @MattGomb,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!




### To highlight:

-  Tests are passing (when running the specific directories in the commands)✔️
-  Nice code organization ✔️ 
-  API is working well✔️ 
-  Good readme ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md [hunk 1]
```diff
@@ -1,24 +1,220 @@
...
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rails db:migrate RAILS_ENV=test
+
+rspec
+```
```
[the remark this answers is not in the document]
Comment (MattGomb) [reply README.md:142]: Commands updated! ✔️

File: README.md [hunk 2]
```diff
@@ -1,24 +1,220 @@
...
+
+  - The app uses various security measures, such as encryption, authentication, and authorization, to protect user data and prevent unauthorized access.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
[the remark this answers is not in the document]
Comment (MattGomb) [reply README.md:218]: License file added ✔️

File: README.md [hunk 3]
```diff
@@ -1,24 +1,220 @@
...
+```
+
+Create Database 
+ - Modify config/database.yml line 6 with your postgres password
+
+```
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
```
[the remark this answers is not in the document]
Comment (MattGomb) [reply README.md:120]: Duplicates removed ✔️

**Comment by MattGomb** [comment]
@Meltrust I just wanted to say that we really appreciate your thoroughness in this review! It is great that you took the time and went into detail, and your additional comments (especially on the front end) are super constructive! Right now I am not going to send for another review, because the front end is still not done, but I am super grateful for the effort you've put into this! 

p.s.: The functionality of the front end is broken most likely because we switched from local storage to session storage, before that everything was working perfectly...

**Review by Meltrust — COMMENTED** [review]
#### Hi @MattGomb,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Tests instructions have been corrected✔️ 
- License has been corrected✔️ 
- Commented code has been removed✔️
- Install steps have been corrected ✔️ 
- Awesome job ✔️  

**I read your note.  Kindly, after your evaluation, please complete the mobile view 💪** 



Cheers and Happy coding!👏👏👏


------

**Comment by Meltrust** [comment]
> @Meltrust I just wanted to say that we really appreciate your thoroughness in this review! It is great that you took the time and went into detail, and your additional comments (especially on the front end) are super constructive! Right now I am not going to send for another review, because the front end is still not done, but I am super grateful for the effort you've put into this!
> 
> p.s.: The functionality of the front end is broken most likely because we switched from local storage to session storage, before that everything was working perfectly...

It's working now.  Great job!

**Pull request merged**